docs/environment/custom-domains.md

```diff
@@ -31,9 +31,10 @@ After validating the domain and the certificate we can now link the custom domai
 - click "Create"
 - enter your domain name, select the certificate you created above and save
 - edit the domain that was created
-- add a "Path mapping" to map the `/` URL (or any URL you want) to your HTTP application in the `Prod` stage, for example:
+- click "Configure API mappings" to add an "API mapping" which maps the `/` URL (or any URL you want) to your HTTP application in the `Dev` stage, for example:
 
   ![](custom-domains-path-mapping.png)
+If you are using Cloudfront:
```

Comment:
I'm not using Cloudfront, and so I couldn't verify if the following 2 instructions show in the new API gateway interface.

Reply:
It's not really related to CloudFront. It's simply that after setting up the "Custom domain", we need to create a DNS entry to point our custom domain to API Gateway.

Check out my other comment, I've suggested to reword this a little bit. The "If you are using Cloudfront:" should be removed then.

```suggestion

```